src/components/format-bytes/format-bytes.test.ts

```diff
@@ -0,0 +1,167 @@
+import { expect, fixture, html, elementUpdated } from '@open-wc/testing';
+import type SlFormatBytes from './format-bytes';
+
+describe('<sl-format-bytes>', () => {
+
+  describe('defaults ', () => {
+
+    it('default properties', async () => {
+      const el = await fixture<SlFormatBytes>(html` <sl-format-bytes></sl-format-bytes> `);
+
+      expect(el.value).to.equal(0);
+      expect(el.unit).to.equal('byte');
+      expect(el.display).to.equal('short');
+      expect(el.lang).to.be.undefined;
+    });
+  });
+
+  describe('bytes', () => {
+
+    it('bytes : short display', async () => {
+      const el = await fixture<SlFormatBytes>(html` <sl-format-bytes></sl-format-bytes> `);
+
+      el.value = 12;
+      await elementUpdated(el);
+      // webkit keeps the lit comment in textContent
+      expect(el.shadowRoot?.textContent?.endsWith('12 byte')).to.be.true;
+
+      el.value = 1200;
+      await elementUpdated(el);
+      // webkit keeps the lit comment in textContent
+      expect(el.shadowRoot?.textContent?.endsWith('1.2 kB')).to.be.true;
+
+      el.value = 1200000;
+      await elementUpdated(el);
+      // webkit keeps the lit comment in textContent
+      expect(el.shadowRoot?.textContent?.endsWith('1.2 MB')).to.be.true;
+
+      el.value = 1200000000;
+      await elementUpdated(el);
+      // webkit keeps the lit comment in textContent
+      expect(el.shadowRoot?.textContent?.endsWith('1.2 GB')).to.be.true;
```

Comment:
Can we loop through these since they're the same for each test? That would remove a lot of boilerplate.

Reply:
yep for sure, was just getting them working, will loop through for sure